environment.yml

```diff
@@ -13,8 +13,12 @@ dependencies:
   - conda-forge::pymdown-extensions=7.1
   - conda-forge::pygments=2.6.1
   - conda-forge::pigz=2.3.4
-  - conda-forge::r-base=3.6.2
   - conda-forge::bc=1.07.1
+  - conda-forge::r-base=3.6.2
+  - conda-forge::r-tidyr=1.1.0
+  - conda-forge::r-optparse=1.6.6
+  - conda-forge::r-ggplot2=3.3.1
+  - conda-forge::r-reshape2=1.4.4
```

Comment:
Mention in the changelog maybe? Though I guess you're adding these as a test for the actions workflows and a PR for these additions will come later..?

Reply:
Ah, yes, sorry. I have changed the CHANGELOG locally but havent pushed the changes. Just updating and testing the whole environment at the mo but will include any additions/updates there too 👍

Reply:
Done now 🙂